README.md

```diff
@@ -98,6 +98,10 @@ filebeat_config: |
   {{filebeat_config_logging}}
 ```
 
+FileBeat templates (a list of templates to install).
```

Comment:
I think it'd be good to show a small example here, since it's not 100% clear exactly how to use this (I understand it's just a list, but an example I think would be quite helpful with this option).